Common/UnitDefinitions/MassConcentration.json

```diff
@@ -0,0 +1,246 @@
+          "Abbreviations": [ "г/м³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerCubicInch",
+      "PluralName": "PoundsPerCubicInch",
+      "BaseUnits": {
+        "L": "Inch",
+        "M": "Pound"
+      },
+      "FromUnitToBaseFunc": "x/3.6127298147753e-5",
+      "FromBaseToUnitFunc": "x*3.6127298147753e-5",
+      "Prefixes": [ "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/in³" ],
+          "AbbreviationsWithPrefixes": [ "kip/in³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerCubicFoot",
+      "PluralName": "PoundsPerCubicFoot",
+      "BaseUnits": {
+        "L": "Foot",
+        "M": "Pound"
+      },
+      "FromUnitToBaseFunc": "x/0.062427961",
+      "FromBaseToUnitFunc": "x*0.062427961",
+      "Prefixes": [ "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/ft³" ],
+          "AbbreviationsWithPrefixes": [ "kip/ft³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "TonnePerCubicMillimeter",
+      "PluralName": "TonnesPerCubicMillimeter",
+      "BaseUnits": {
+        "L": "Millimeter",
+        "M": "Tonne"
+      },
+      "FromUnitToBaseFunc": "x/1e-12",
+      "FromBaseToUnitFunc": "x*1e-12",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "t/mm³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "TonnePerCubicCentimeter",
+      "PluralName": "TonnesPerCubicCentimeter",
+      "BaseUnits": {
+        "L": "Centimeter",
+        "M": "Tonne"
+      },
+      "FromUnitToBaseFunc": "x/1e-9",
+      "FromBaseToUnitFunc": "x*1e-9",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "t/cm³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "TonnePerCubicMeter",
+      "PluralName": "TonnesPerCubicMeter",
+      "BaseUnits": {
+        "L": "Meter",
+        "M": "Tonne"
+      },
+      "FromUnitToBaseFunc": "x/0.001",
+      "FromBaseToUnitFunc": "x*0.001",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "t/m³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SlugPerCubicFoot",
+      "PluralName": "SlugsPerCubicFoot",
+      "BaseUnits": {
+        "L": "Foot",
+        "M": "Slug"
+      },
+      "FromUnitToBaseFunc": "x*515.378818",
+      "FromBaseToUnitFunc": "x*0.00194032033",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "slug/ft³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "GramPerLiter",
+      "PluralName": "GramsPerLiter",
+      "FromUnitToBaseFunc": "x/1",
+      "FromBaseToUnitFunc": "x*1",
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/l"]
```

Comment:
I would suggest using capital `L` for liter as I believe that is the most common form? I do see we are not very consistent in our existing units and found several of both lower and uppercase.

Reply:
fixed for g/l, g/dl, g/ml & kg/l